strimzi/assets/dashboards/overview.json

```
@@ -0,0 +1,1850 @@
+                                        {
+                                            "name": "query1",
+                                            "data_source": "metrics",
+                                            "query": "avg:strimzi.cluster_operator.resources.paused{host:$host.value,endpoint:$endpoint.value} by {endpoint,host,kind,namespace,selector}",
+                                            "aggregator": "avg"
+                                        }
+                                    ],
+                                    "formulas": [
+                                        {
+                                            "formula": "query1"
+                                        }
+                                    ]
+                                }
+                            ]
+                        },
+                        "layout": {
+                            "x": 8,
+                            "y": 3,
+                            "width": 4,
+                            "height": 3
+                        }
+                    },
+                    {
+                        "id": 4449867539733798,
+                        "definition": {
+                            "title": "Reconciliation Duration",
+                            "title_size": "16",
+                            "title_align": "left",
+                            "show_legend": true,
+                            "legend_layout": "vertical",
+                            "legend_columns": [
+                                "avg",
+                                "min",
+                                "max",
+                                "value",
+                                "sum"
+                            ],
+                            "time": {},
+                            "type": "timeseries",
+                            "requests": [
+                                {
+                                    "formulas": [
+                                        {
+                                            "number_format": {
+                                                "unit": {
+                                                    "type": "canonical_unit",
+                                                    "unit_name": "second"
+                                                }
+                                            },
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "queries": [
+                                        {
+                                            "name": "query1",
+                                            "data_source": "metrics",
+                                            "query": "avg:strimzi.cluster_operator.reconciliations.duration_seconds.count{$host, $endpoint} by {endpoint,host,kind,namespace,selector}.as_count()"
```

```suggestion
                                            "query": "sum:strimzi.cluster_operator.reconciliations.duration_seconds.count{$host, $endpoint} by {endpoint,host,kind,namespace,selector}.as_count()"
```

What are you intending the show here? Based on the query it seems like the average reconciliation duration in seconds across endpoint,host,kind,namespace,selector.

If we use sum then we get the sum of the duration across endpoint,host,kind,namespace,selector

---

Changing between `avg` and `sum` seemed to be the same. I wanted to show the duration of reconciliations for each configuration of the Strimzi cluster every second. In theory, would `sum` make more sense than `avg` for this?

---

I think that is because you only have one dataset across all those tags. I think sum might be better here. You also use it in your other graphs as well.